src/core/gui/dialog/toolbarCustomize/ToolbarDragDropHelper.cpp

```diff
@@ -39,7 +39,7 @@ auto gdk_context_set_icon_from_image(GdkDragContext* ctx, GtkWidget* widget) ->
         }
         case GTK_IMAGE_SURFACE: {
             cairo_surface_t* surface{};
-            g_object_get(G_OBJECT(image), "surface", &surface);
+            g_object_get(G_OBJECT(image), "surface", &surface, nullptr);
```

Comment:
This behavior is potentially different because previously this sentinel value wasn't added to denote the end of the variadic arguments. So I guess the behavior was previously UB? According to the [docs](https://docs.gtk.org/gobject/method.Object.get.html) it specifies to add `NULL`, but Clang recommends passing in `nullptr` and [this](https://stackoverflow.com/questions/65279075/is-it-safe-to-pass-nullptr-to-c-function) answer seems to indicate that it is OK to pass `nullptr` (just covering all bases here)

Reply:
Yes, I was probably UB and segfault prone. Good catch!

Reply:
Actually, now that I think about it, these specific changes should go into the release branch (this could fix some segfaults people might have experienced).
Would you mind taking them out of this PR and into a new PR targetted at release-1.1?